### ISSUE TITLE
Could not open library 'C:/Ruby26/lib/ruby/gems/2.6.0/gems/fast_excel-0.2.6/libxlsxwriter/lib/libxlsxwriter.so'

### ISSUE THREAD

**Issue by Rajagopalan-M** [Paxa/fast_excel#61]
Hi,I am using windows OS, I have installed the gem and started running your given example, but I am getting the following error

```
C:\Ruby26\bin\ruby.exe C:/A/RubyAutomation/AppData/Example.rb
Traceback (most recent call last):
        2: from C:/A/RubyAutomation/AppData/Example.rb:2:in `<main>'
        1: from C:/Ruby26/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
C:/Ruby26/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- fast_excel (LoadError)
        10: from C:/A/RubyAutomation/AppData/Example.rb:2:in `<main>'
         9: from C:/Ruby26/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:34:in `require'
         8: from C:/Ruby26/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `rescue in require'
         7: from C:/Ruby26/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `require'
         6: from C:/Ruby26/lib/ruby/gems/2.6.0/gems/fast_excel-0.2.6/lib/fast_excel.rb:1:in `<top (required)>'
         5: from C:/Ruby26/lib/ruby/gems/2.6.0/gems/fast_excel-0.2.6/lib/fast_excel.rb:1:in `require_relative'
         4: from C:/Ruby26/lib/ruby/gems/2.6.0/gems/fast_excel-0.2.6/lib/fast_excel/binding.rb:5:in `<top (required)>'
         3: from C:/Ruby26/lib/ruby/gems/2.6.0/gems/fast_excel-0.2.6/lib/fast_excel/binding.rb:14:in `<module:Libxlsxwriter>'
         2: from C:/Ruby26/lib/ruby/gems/2.6.0/gems/ffi-1.12.2-x86-mingw32/lib/ffi/library.rb:99:in `ffi_lib'
         1: from C:/Ruby26/lib/ruby/gems/2.6.0/gems/ffi-1.12.2-x86-mingw32/lib/ffi/library.rb:99:in `map'
 (LoadError)b/ruby/gems/2.6.0/gems/ffi-1.12.2-x86-mingw32/lib/ffi/library.rb:145:in `block in ffi_lib': Could not open library 'C:/Ruby26/lib/ruby/gems/2.6.0/gems/fast_excel-0.2.6/libxlsxwriter/lib/libxlsxwriter.so': The specified module could not be found.

```

**Comment by preetpalS** [comment]
I also have the same issue (using git master):

```
C:\Users\preet\Projects\OpenSource\fast_excel>bundle exec rake test
ansi: 'gem install win32console' to use color on Windows
Traceback (most recent call last):
        14: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:5:in `<main>'
        13: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:5:in `select'
        12: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:17:in `block in <main>'
        11: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib/rake/rake_test_loader.rb:17:in `require'
        10: from C:/Users/preet/Projects/OpenSource/fast_excel/test/auto_width_test.rb:1:in `<top (required)>'
         9: from C:/Users/preet/Projects/OpenSource/fast_excel/test/auto_width_test.rb:1:in `require_relative'
         8: from C:/Users/preet/Projects/OpenSource/fast_excel/test/test_helper.rb:15:in `<top (required)>'
         7: from C:/Users/preet/Projects/OpenSource/fast_excel/test/test_helper.rb:15:in `require_relative'
         6: from C:/Users/preet/Projects/OpenSource/fast_excel/lib/fast_excel.rb:1:in `<top (required)>'
         5: from C:/Users/preet/Projects/OpenSource/fast_excel/lib/fast_excel.rb:1:in `require_relative'
         4: from C:/Users/preet/Projects/OpenSource/fast_excel/lib/fast_excel/binding.rb:5:in `<top (required)>'
         3: from C:/Users/preet/Projects/OpenSource/fast_excel/lib/fast_excel/binding.rb:14:in `<module:Libxlsxwriter>'
         2: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/ffi-1.13.1-x64-mingw32/lib/ffi/library.rb:99:in `ffi_lib'
         1: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/ffi-1.13.1-x64-mingw32/lib/ffi/library.rb:99:in `map'
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/ffi-1.13.1-x64-mingw32/lib/ffi/library.rb:145:in `block in ffi_lib': Could not open library 'C:/Users/preet/Projects/OpenSource/fast_excel/libxlsxwriter/lib/li (LoadError)so': The specified module could not be found.
rake aborted!
Command failed with status (1)

Tasks: TOP => test
(See full trace by running task with --trace)

C:\Users\preet\Projects\OpenSource\fast_excel>

```

**Comment by Rajagopalan-M** [comment]
@preetpalS I think they are not maintaining this gem.

**Comment by Rajagopalan-M** [comment]
@Paxa @preetpalS  Hi, It looks like this issue is fixed but when I installed and ran today, Once again, I am getting the same error
```

Traceback (most recent call last):
        2: from C:/A/Rajagopalan/AppData/Example.rb:1:in `<main>'
        1: from C:/Ruby27/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
C:/Ruby27/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require': cannot load such file -- fast_excel (LoadError)
        10: from C:/A/Rajagopalan/AppData/Example.rb:1:in `<main>'
         9: from C:/Ruby27/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:156:in `require'
         8: from C:/Ruby27/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `rescue in require'
         7: from C:/Ruby27/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `require'
         6: from C:/Ruby27/lib/ruby/gems/2.7.0/gems/fast_excel-0.3.0/lib/fast_excel.rb:1:in `<top (required)>'
         5: from C:/Ruby27/lib/ruby/gems/2.7.0/gems/fast_excel-0.3.0/lib/fast_excel.rb:1:in `require_relative'
         4: from C:/Ruby27/lib/ruby/gems/2.7.0/gems/fast_excel-0.3.0/lib/fast_excel/binding.rb:5:in `<top (required)>'
         3: from C:/Ruby27/lib/ruby/gems/2.7.0/gems/fast_excel-0.3.0/lib/fast_excel/binding.rb:14:in `<module:Libxlsxwriter>'
         2: from C:/Ruby27/lib/ruby/gems/2.7.0/gems/ffi-1.14.2-x86-mingw32/lib/ffi/library.rb:99:in `ffi_lib'
         1: from C:/Ruby27/lib/ruby/gems/2.7.0/gems/ffi-1.14.2-x86-mingw32/lib/ffi/library.rb:99:in `map'
C:/Ruby27/lib/ruby/gems/2.7.0/gems/ffi-1.14.2-x86-mingw32/lib/ffi/library.rb:145:in `block in ffi_lib': Could not open library 'C:/Ruby27/lib/ruby/gems/2.7.0/gems/fast_excel-0.3.0/libxlsxwriter/lib/libxlsxwriter.so': The specified module could not be found.\r (LoadError)
```

**Comment by preetpalS** [comment]
@Rajagopalan-M The fix is not in version 0.3.0 of the gem (it was released before fix applied in the master branch). Install the gem from the master branch (use bundler for your project and lock the gem version down to a specific commit).

**Comment by Paxa** [comment]
I released latest changes, please try and if problem still exists then re-open issue
